example/cmd/device-simple/Attribution.txt

```diff
@@ -130,4 +130,7 @@ go-playground/validator/v10 (MIT) https://github.com/go-playground/validator/v10
 https://github.com/go-playground/validator/blob/master/LICENSE
 
 leodido/go-urn (MIT) https://github.com/leodido/go-urn
-https://github.com/leodido/go-urn/blob/master/LICENSE
\ No newline at end of file
+https://github.com/leodido/go-urn/blob/master/LICENSE
+
+stretchr/objx (MIT) https://github.com/stretchr/objx
+https://github.com/stretchr/objx/blob/master/LICENSE
```

Comment:
Not sure why this just now was identified in this PR, but it is required for PR to pass verify checks.